src/main/java/org/maxgamer/quickshop/shop/ContainerShop.java

```diff
@@ -734,7 +750,10 @@ public void onLoad() {
                 }
             }
         }
-        this.checkDisplay();
+        try {
+            this.checkDisplay();
+        } catch (Exception ignored) {
```

Comment:
Why we need to make a try-catch block?
Any comment for this?

Reply:
A bug in VirtualDisplayItem, when an error thrown in VDI, the exception will be caught in ShopLoader and failed to load the shop

Reply:
What exception it throws?